src/main/java/decodes/dbeditor/PlatformListPanel.java

```diff
@@ -80,26 +277,38 @@ public Platform getSelectedPlatform()
 		return platformSelectPanel.getSelectedPlatform();
 	}
 
+	/**
+	 * Get the selected Platform from the filtered list
+	 * 
+	 * @return Platform
+	 */
+	public Platform getSelectedItem() {
+		int rowNumb = this.platformSelectPanel.getPlatformListTable().getSelectedRow();
+		int rowModel = this.platformSelectPanel.getPlatformListTable().convertRowIndexToModel(rowNumb);
+		PlatformSelectTableModel model = (PlatformSelectTableModel) platformSelectPanel.getPlatformListTable()
+				.getModel();
+		Platform platform = model.getPlatformAt(rowModel);
+		return platform;
+	}
+
 	/** Called when the 'Open' button is pressed. */
-	public void openPressed()
-	{
-		Platform p = platformSelectPanel.getSelectedPlatform();
+	public void openPressed() {
+		int rows = this.platformSelectPanel.getPlatformListTable().getRowCount();
+		// System.out.println("Open rows: " + rows);
+		Platform p = getSelectedItem();
+		// Platform p = platformSelectPanel.getSelectedPlatform();
 		if (p == null)
 			TopFrame.instance().showError(
-				dbeditLabels.getString("PlatformListPanel.selectOpen"));
-		else
-		{
-			try
-			{
+					dbeditLabels.getString("PlatformListPanel.selectOpen"));
+		else {
+			try {
 				p.read();
 				doOpen(p);
-			}
-			catch(Exception ex)
-			{
+			} catch (Exception ex) {
```

Comment:
catch should be on next line.

the try and else above should have { on the next line.